aws-precise-staging-1/main.tf

```diff
@@ -44,14 +45,15 @@ module "aws_asg_org" {
   cyclist_scale = "web=1:Hobby"
   cyclist_version = "v0.1.0"
   env = "${var.env}"
+  env_short = "${var.env_short}"
   heroku_org = "${var.aws_heroku_org}"
   index = "${var.index}"
   security_groups = "${module.aws_az_1b.workers_org_security_group_id},${module.aws_az_1e.workers_org_security_group_id}"
   site = "org"
   syslog_address = "${var.syslog_address}"
   worker_ami = "${var.worker_ami}"
   worker_asg_max_size = "3"
-  worker_asg_min_size = "1"
+  worker_asg_min_size = "0"
```

Comment:
rather, will this make it so that a whole instance needs to start up before a staging build can be run?